pages/help-center/errors/serialization.mdx

```diff
@@ -1,53 +1,101 @@
 ---
-title: Serialization errors
-description: Learn about serialization errors as well as troubleshooting best practices for serialization errors in our all-in-one documentation page at Memgraph. 
+title: Conflicting transaction and serialization error
```

Comment:
Do you want this title to be on the sidebar menu as well (it's a bit long maybe). Currently Serialization is still on the sidebar menu, but if users are seeing a Conflicting transaction error, it might be better to put that on sidebar. Wdyt?

Reply:
Agreed